packages/simpleserialize.com/webpack.config.js

```diff
@@ -20,6 +20,12 @@ const config = {
   },
   module: {
     rules: [
```

Comment:
Adding this rule was recommended in https://github.com/graphql/graphql-js/issues/2721#issuecomment-723008284 and fixes the following build error. It is also import to add it as first rule, else it won't work.

```
ERROR in ../../node_modules/@lodestar/params/lib/index.js 26:68-75
Module not found: Error: Can't resolve 'process/browser' in '/home/nico/projects/ethereum/ssz/node_modules/@lodestar/params/lib'
Did you mean 'browser.js'?
BREAKING CHANGE: The request 'process/browser' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
resolve 'process/browser' in '/home/nico/projects/ethereum/ssz/node_modules/@lodestar/params/lib'
  Parsed request is a module
  using description file: /home/nico/projects/ethereum/ssz/node_modules/@lodestar/params/package.json (relative path: ./lib)
    Field 'browser' doesn't contain a valid alias configuration
    resolve as module
      /home/nico/projects/ethereum/ssz/node_modules/@lodestar/params/lib/node_modules doesn't exist or is not a directory
      /home/nico/projects/ethereum/ssz/node_modules/@lodestar/params/node_modules doesn't exist or is not a directory
      /home/nico/projects/ethereum/ssz/node_modules/@lodestar/node_modules doesn't exist or is not a directory
      /home/nico/projects/ethereum/ssz/node_modules/node_modules doesn't exist or is not a directory
      looking for modules in /home/nico/projects/ethereum/ssz/node_modules
        existing directory /home/nico/projects/ethereum/ssz/node_modules/process
          using description file: /home/nico/projects/ethereum/ssz/node_modules/process/package.json (relative path: .)
            using description file: /home/nico/projects/ethereum/ssz/node_modules/process/package.json (relative path: ./browser)
              Field 'browser' doesn't contain a valid alias configuration
              /home/nico/projects/ethereum/ssz/node_modules/process/browser doesn't exist
      /home/nico/projects/ethereum/node_modules doesn't exist or is not a directory
      /home/nico/projects/node_modules doesn't exist or is not a directory
      /home/nico/node_modules doesn't exist or is not a directory
      /home/node_modules doesn't exist or is not a directory
      /node_modules doesn't exist or is not a directory
```